### DISCUSSION TITLE
在初始化安装后nb run 报错

### DISCUSSION THREAD

**Discussion by Cans518** [nonebot/discussions#385] [问答]
报错信息：
![图片](https://github.com/nonebot/discussions/assets/52856665/40bec4ab-d19a-4a69-a493-475980183975)
系统版本：Ubuntu 22.04.2 LTS 64位
Python版本：Python 3.10.6 (main, May 29 2023, 11:10:38) [GCC 11.3.0] on linux
<img width="353" alt="image" src="https://github.com/nonebot/discussions/assets/52856665/fc97cb59-5353-41f6-9c65-dc47649cd17e">

**Comment by yanyongyu** [comment]
你在初始化的时候选择了错误的驱动器，请修改dotenv文件内的driver配置

**Reply by Cans518** [reply]
请问应该使用哪一个驱动器呢？
仅仅是使用QQ和QQ频道机器的话
谢谢啦。

**Reply by yanyongyu** [reply]
`~fastapi+~aiohttp`或者`~fastapi+~httpx+~websockets`
